test_get_number_of_char (__main__.TestExercice) ... ok
test_get_number_of_words (__main__.TestExercice) ... ok
test_pair (__main__.TestExercice) ... FAIL
test_remove_third_char (__main__.TestExercice) ... ok
test_replace_char (__main__.TestExercice) ... ERROR

======================================================================
ERROR: test_replace_char (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 43, in test_replace_char
    output = [exercice.replace_char(v[0], v[1], v[2]) for v in values]
  File "test_exercice.py", line 43, in <listcomp>
    output = [exercice.replace_char(v[0], v[1], v[2]) for v in values]
  File "/home/runner/work/2021a-c01-ch4-exercices-mipie/2021a-c01-ch4-exercices-mipie/exercice.py", line 24, in replace_char
    caract[6] = "z"
IndexError: list assignment index out of range

======================================================================
FAIL: test_pair (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 18, in test_pair
    self.assertListEqual(
AssertionError: Lists differ: [True, True, True] != [True, False, False]

First differing element 1:
True
False

- [True, True, True]
+ [True, False, False] : Mauvaise identification de la parit√© de la longueur de la chaine

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (failures=1, errors=1)
